app/routes/docs.$lang.$ref.tsx

```diff
@@ -119,24 +119,27 @@ export default function DocsLayout() {
         }
       >
         <InnerContainer>
-          <NavMenuDesktop />
-          <div
-            ref={docsContainer}
-            className={cx(
-              // add scroll margin to focused elements so that they aren't
-              // obscured by the sticky header
-              "[&_*:focus]:scroll-mt-[8rem] lg:[&_*:focus]:scroll-mt-[5rem]",
-              "min-h-[80vh]",
-              "lg:ml-72 lg:pl-6 xl:pl-10 2xl:pl-12",
-              !changingVersions && navigating
-                ? "opacity-25 transition-opacity delay-300"
-                : "",
-            )}
-          >
-            <Outlet />
-          </div>
-          <div className="min-w-0 pt-8 sm:pt-10 lg:ml-72 lg:pl-6 lg:pt-12 xl:pl-10 2xl:pl-12">
-            <Footer />
+          <div className="block lg:flex">
+            <NavMenuDesktop />
+            <div
+              ref={docsContainer}
+              className={cx(
+                // add scroll margin to focused elements so that they aren't
+                // obscured by the sticky header
+                "[&_*:focus]:scroll-mt-[8rem] lg:[&_*:focus]:scroll-mt-[5rem]",
+                // Account for the left navbar
+                "min-h-[80vh] lg:ml-3 lg:w-[calc(100%-theme(spacing.72))]",
```

Comment:
Seems like you don't need width here, one less magic number to keep track of.
```suggestion
                "min-h-[80vh] lg:ml-3 lg:flex-1",
```

Reply:
Just kidding, I'm back and trying to get this in real quick

I definitely love the idea of not having another magic number! However, it seems like you can get into a weird state here with a squished left nav.

<img width="1047" alt="image" src="https://github.com/remix-run/remix-website/assets/12396812/d8a44d8f-9c69-4c92-af5d-230615b2d1aa">

If I make the left nave have a `min-w-72`, then at this size some of the doc gets cut off and you can't even horizontally scroll to it.

I think I'm just going to take the same approach with the height that you shared, but if you have a way to account for all these issues I'm all ears!